pym/bob/cmds/build.py

```diff
@@ -887,6 +909,13 @@ def commonBuildDevelop(parser, argv, bobRoot, develop):
     elif len(results) > 1:
         print("Build results are in:\n  ", "\n   ".join(results))
 
+    print("checkoutSteps: " + str(builder.getStatistic()['checkoutSteps']))
+    print("buildSteps:    " + str(builder.getStatistic()['buildSteps']))
+    print("packageSteps:  " + str(builder.getStatistic()['packageSteps']))
+    print("scms: " + str(builder.getStatistic()['scms']))
+    print(" -> overridden: " + str(builder.getStatistic()['scmOverrides']))
+    print("   -> packages: " + str(builder.getStatistic()['scmOverriddenPackages']))
```

Comment:
Well - I wasn't happy with my solution but had no better idea... :wink: 

I like your suggestions - they show some (maybe useful) informations while not over-rolling the user.